src/features/firstLogin/PrivacyPolicy/PrivacyPolicy.native.test.tsx

```diff
@@ -12,6 +18,25 @@ jest.mock('features/auth/settings', () => ({
   useAppSettings: jest.fn(() => mockSettings()),
 }))
 
+const defaultUseCookies = {
+  cookiesConsent: {
+    mandatory: COOKIES_BY_CATEGORY.essential,
+    accepted: ALL_OPTIONAL_COOKIES,
+    refused: [],
+  },
+  setCookiesConsent: (_: Consent) => new Promise<void>(() => null),
+  setUserId: (_: number) => new Promise<void>(() => null),
```

Comment:
j'imagine que ces fonctions sont présentent juste pour le typage

si c'est le cas, je trouve le code difficillement lisible

je recommande d'utiliser `async`
```suggestion
  setCookiesConsent: async (_: Consent) => {},
  setUserId: async (_: number) => {},
```

ou encore plus simple

```suggestion
  setCookiesConsent: jest.fn(),
  setUserId: jest.fn(),
```

Reply:
Bizarrement j'ai pas pensé à `jest.fn()` 🤔